docs/api.asciidoc

```diff
@@ -0,0 +1,159 @@
+
+[source,python]
+----
+from elasticapm import Client
+
+client = Client({'APP_NAME': 'example'}, **defaults)
+----
+
+[break]
+===== Arguments
+
+ * `config`: A dictionary, with key/value configuration. For the possible configuration keys, see <<configuration, Configuration>>.
+ * `**defaults`: default values for configuration. These can be omitted in most cases, and take the least precedence.
+
+[[error-api]]
+=== Error API
+
+[break]
+[[client-api-capture-exception]]
+==== `Client.capture_exception()`
+
+Captures an exception object:
+
+[source,python]
+----
+try:
+    x = int("five")
+except ValueError:
+    client.capture_exception()
+----
+
+[break]
+===== Arguments
+
+ * `exc_info`: A `(type, value, traceback)` tuple as returned by https://docs.python.org/3/library/sys.html#sys.exc_info[`sys.exc_info()`]. If not provided, it will be captured automatically.
+ * `date`: A `datetime.datetime` object representing the occurrence time of the error. If left empty, it defaults to `datetime.datetime.utcnow()`.
+ * `context`: A dictionary with contextual information. This dictionary must follow the 
+    https://www.elastic.co/guide/en/apm/server/current/error-api.html#error-context-schema[Context] schema definition.
+ * `custom`: A dictionary of custom data you want to attach to the event.
+
+Returns the id of the error as a string.
+
+[break]
+[[client-api-capture-exception]]
+==== `Client.capture_message()`
+
+Captures a message with optional added contextual data.
+
+[source,python]
+----
+client.capture_message('Billing process succeeded.')
+----
+
+[break]
+===== Arguments
+
+ * `message`: The message as a string.
+ * `param_message`: Alternatively, a parametrized message as a dictionary.
+    The dictionary contains two values: `message`, and `params`.
+    This allows the APM server to group messages together that have share the same
+    parametrized message. Example:
++
+[source,python]
+----
+client.capture_message(param_message={
+    'message': 'Billing process for %s succeeded. Amount: %s',
+    'params': (customer.id, order.total_amount),
+})
+----
++
+ * `stack`: If set to `True` (the default), a stacktrace from the call site will be captured.
+ * `exc_info`: A `(type, value, traceback)` tuple as returned by https://docs.python.org/3/library/sys.html#sys.exc_info[`sys.exc_info()`]. If not provided, it will be captured automatically, if `capture_message()` was called in an `except` block.
+ * `date`: A `datetime.datetime` object representing the occurrence time of the error. If left empty, it defaults to `datetime.datetime.utcnow()`.
+ * `context`: A dictionary with contextual information. This dictionary must follow the 
+    https://www.elastic.co/guide/en/apm/server/current/error-api.html#error-context-schema[Context] schema definition.
+ * `custom`: A dictionary of custom data you want to attach to the event.
+
+Returns the id of the message as a string.
+
+[[transaction-api]]
+=== Transaction API
+
+[break]
```

Comment:
Although the usage is straightforward, it would be nice to also have an example in for the `transaction` cmds, as you also have examples for `client` and `error` related usage.